api_admin_panel/app/components/output.py

```diff
@@ -0,0 +1,98 @@
+import reflex as rx
+
+from app.states.queries import QueryAPI
+from app.components.drawer import render_drawer
+
+
+def create_table_header(title: str):
+    return rx.table.column_header_cell(title)
+
+
+def create_query_rows(data: dict[str, str]):
+    def fill_rows_with_data(data_):
+        return rx.table.cell(
+            f"{data_[1]}",
+            on_click=QueryAPI.display_selected_row(data),
+            cursor="pointer",
+        )
+
+    return rx.table.row(
+        rx.foreach(data, fill_rows_with_data),
+        _hover={
```

Comment:
For the hover color I think we can is rx.color with shade 10 https://www.radix-ui.com/colors/docs/palette-composition/understanding-the-scale, its meant for hovered element colro

Reply:
The URL is a good reference - I went with `shade=4` because `shade=10` didn't look as good. If you want it at `10` let me know.